package-lock.json

```diff
@@ -28598,14 +28598,14 @@
       }
     },
     "node_modules/electron": {
-      "version": "26.6.8",
-      "resolved": "https://registry.npmjs.org/electron/-/electron-26.6.8.tgz",
-      "integrity": "sha512-nuzJ5nVButL1jErc97IVb+A6jbContMg5Uuz5fhmZ4NLcygLkSW8FZpnOT7A4k8Saa95xDJOvqGZyQdI/OPNFw==",
+      "version": "29.0.0",
+      "resolved": "https://registry.npmjs.intuit.com:443/artifactory/api/npm/npm-intuit/electron/-/electron-29.0.0.tgz",
```

Comment:
@jeremy-croff i think we shouldn't use intuit artifactory. Please install electron from `npm`

Reply:
@rushatgabhane Amazing catch, thank you. I do point to intuit for other projects and will fix this.

Reply:
@rushatgabhane I have addressed this